packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-number/example.html

```diff
@@ -4,220 +4,56 @@ <h1>gux-form-field-number</h1>
 
 <h2>Examples</h2>
 <form onchange="notify(event)" oninput="notify(event)">
-  <fieldset>
-    <legend>Clearable</legend>
-
-    <gux-form-field-number>
-      <input slot="input" type="number" name="c-1" value="10" />
-      <label slot="label">Default</label>
```

Comment:
I will revert the changes in this file, I am just using it as an example so you dont have to make one yourself.